src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -126,6 +132,33 @@ private Tenant saveTenant(UUID consortiumId, Tenant tenantDto) {
     return converter.convert(savedTenant, Tenant.class);
   }
 
+  private UserTenant postUserTenant(UserTenant userTenant) {
+    log.info("Creating userTenant with dummy user with id {}.", userTenant.getId());
+    userTenantsClient.postUserTenant(userTenant);
+    return userTenant;
+  }
+
+  /*
+    Dummy user will be used to support cross-tenant requests checking in mod-authtoken,
+    if user-tenant table contains some record in institutional tenant - it means mod-consortia enabled for
+    this tenant and will allow cross-tenant request.
+
+    @param tenantId tenant id
+  */
+  private void saveDummyUser(String tenantId) {
+    UserTenant userTenant = createUserTenantWithDummyUser(tenantId);
+    postUserTenant(userTenant);
+  }
+
+  private UserTenant createUserTenantWithDummyUser(String tenantId) {
```

Comment:
lets remain only this method createUserTenantWithDummyUser(), postUserTenant, saveDummyUser methods are not needed.
Java doc also should be moved to this method

Reply:
fixed